src/components/core/Bookmark/BookmarkIcon.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { func } from "prop-types";
+
+import { BookmarkHollowIcon } from "assets/icons/BookmarkHollow";
+import { BookmarkFilledIcon } from "assets/icons/BookmarkFilled";
+
+const Container = styled.div`
+  cursor: pointer;
+  display: flex;
+  margin-left: 20px;
+  height: 40px;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  :hover {
+    background-color: #f6f6f7;
+  }
+`;
+
+const BookmarkIcon = ({ isBookmarked, handleClick }) => {
```

Comment:
I don't like this approach. You have a component called `BookmarkIcon`, but in fact, it's not an icon. Icons should be placed in the `assets` directory. I wouldn't put this logic in a separate directory imho.